src/components/views/donate/EstimatedMatchingToast.tsx

```diff
@@ -88,13 +88,26 @@ const EstimatedMatchingToast: React.FC<IEstimatedMatchingToast> = ({
 			)} ${allocatedFundUSDPreferred ? '' : ` ${allocatedTokenSymbol}`}`
 		: '---';
 
+	const formatMinUSDValue = (
+		num: number | undefined,
+		precision: number = 2,
+	) => {
+		if (!num) return '0';
+
+		if (num >= 1 || num < 0.01) {
+			return num.toString();
+		}
+
+		return num.toFixed(precision);
+	};
+
 	const bottomText = isAboveMinValidUsdValue
 		? formatMessage({ id: 'page.donate.matching_toast.bottom_valid' })
 		: formatMessage({
 				id: 'page.donate.matching_toast.bottom_invalid_p1',
 			}) +
 			' $' +
-			minimumValidUsdValue +
+			formatMinUSDValue(minimumValidUsdValue) +
```

Comment:
I suggest using `formatBalance(minimumValidUsdValue)` instead

Reply:
@RamRamez I made that change, but I think `minimumValidUsdValue` can be also more less than `0.000001` so we will see `0.00` on FE, I think we should either add some min input validation on BE or WDYT?

Reply:
@Meriem-BM It's USD value not token amount. I think setting `minimumValidUsdValue` less than 0.000001 USD doesn't make sense because it's almost zero.